salt/crio/crio.conf.jinja

```diff
@@ -0,0 +1,160 @@
+
+# The "crio" table contains all of the server options.
+[crio]
+
+# root is a path to the "root directory". CRIO stores all of its data,
+# including container images, in this directory.
+root = "{{pillar['crio']['dirs']['root']}}"
+
+# run is a path to the "run directory". CRIO stores all of its state
+# in this directory.
+runroot = "{{pillar['crio']['dirs']['runroot']}}"
```

Comment:
Minor: here as well as on `root`, we usually leave blank spaces between the double braces.